lib/wrappers/transform-node.ts

```diff
@@ -18,7 +17,13 @@ export default class TransformNodeWrapper extends NodeWrapper {
       outputPath: this.outputPath,
       cachePath: this.cachePath,
     });
+    //adding ts-ignore for now. Should be removed later when new version of broccoli-node-api is released
     this.callbackObject = this.nodeInfo.getCallbackObject();
+    // @ts-ignore
+    if (this.nodeInfo.fsFacade) {
+      // @ts-ignore
+      this.nodeInfo.setFSFacade();
```

Comment:
Should we pass the `input` and `output` arguments to this function?

Reply:
Changed it, now passes `input` and `output`